server/command.go

```diff
@@ -62,7 +62,7 @@ func (p *Plugin) ExecuteCommand(c *plugin.Context, args *model.CommandArgs) (*mo
 	switch command {
 	case "add":
 		handler = p.runAddCommand
-	case "list":
+	case "list", "":
```

Comment:
Fascinating...

1) @iomodo I was surprised to see this actually work, apparently args.Command comes in with a space appended at the end. We can argue that the plugin should be `strings.TrimSpace`-ing the command, but that's besides the point. This is probably a bug/undesired feature in the server code somewhere. normalizing the command?
2) @thapakazi I recommend not relying on the trailing " ", and having an empty stringArgs[1]; if anything that's a bug; the right place to insert `p.runListCommand` is line 55; I also recommend adding a `strings.TrimSpace` around `args.Command` in line 52.

Reply:
@levb there are 2 problems:
1. Current implementation of the autocomplete appends trailing space automatically when you type enter after a trigger word. So if you want to type just `/todo` you have to delete the space.
2. Server code assumes there will be a trailing space after the trigger word. There's already a [PR](https://github.com/mattermost/mattermost-server/pull/13646) changing that behavior.